Makefile

```diff
@@ -14,6 +14,12 @@ install :
 	cd cli; \
 	make install
 
+vc:
+	make -C virtcontainers
+
+install-all:
+	make -C cli install
+	make -C virtcontainers install
```

Comment:
The artifacts in virtcontainers are for testing purpose. I don't think we want to package and ship them globally. Maybe we can add `install-vc` that installs the binaries from virtcontainers directory and the test script can make use of it when necessary?

Reply:
@bergwolf Missed this comment. I have added an `install-vc` target for the test binaries.